src/analysis/Environments/Environments.ts

```diff
@@ -579,15 +579,13 @@ export const Environments: React.FC<EnvironmentsProps> = ({ nav = undefined }) =
     return getDetailsPopup(runtimeName, cloudContext?.cloudResource, disk, creator, workspace?.workspaceId);
   };
 
-  const renderDeleteButton = (resourceType, resource) => {
+  const renderDeleteButton = (resourceType: 'app' | 'runtime', resource) => {
```

Comment:
This was only called once, and the `resourceType` argument is either 'app' or 'runtime'. It looks like the logic for `resourceType === 'disk'` was unused.